lib/python-beta/src/bailo/core/agent.py

```diff
@@ -7,10 +7,12 @@ class Agent:
     def __init__(self):
         self.get = requests.get
         self.patch = requests.patch
-        self.post = requests.post
         self.put = requests.put
         self.delete = requests.delete
 
+    def post(self, *args, **kwargs):
```

Comment:
a) Not all of our post requests send and receive JSON (e.g. post file)
b) Instead of setting this, you should likely be using the `json` key on a `requests.post`?